### DISCUSSION TITLE
Changes aren't reflected in UI

### DISCUSSION THREAD

**Discussion by andrewdavis12** [usebruno/bruno#2721] [Q&A] [answered]
Recently, any time that I make any changes to my collection, be it adding a new request, deleting a request, saving changes, etc. the action will complete but nothing in my Bruno UI will update accordingly. So if I delete a request, it doesn't go anywhere. If I add a new request, it will ask me for the details of the request, but when I save, it's nowhere in the UI. If I close Bruno completely and re-open, then the changes are reflected. Anyone seen anything like this or have ideas on where I can find the underlying issue? 

I have updated to the latest, tried a different version, tried the homebrew version, no luck. Perhaps something with my collection even though it's small (only a couple dozen requests), but with no errors or anything I don't know how to troubleshoot further. 

Edit: This is on an apple silicon Mac

**Comment by Its-treason** [comment]
Sounds like a weird issue. Could you check if there are any error in the chrome console (`ctrl`+`shift`+`i`) or run Bruno from a terminal to see if it logs any errors?

**Reply by andrewdavis12** [reply]
No errors when using Bruno CLI. Inspecting the chrome console from Bruno, I see two errors immediately at startup. 
![CleanShot 2024-07-29 at 16 00 04](https://github.com/user-attachments/assets/29c80dff-d40d-4858-b5d1-ca591e8d168e)

which isn't abnormal from looking at my other Chrome usage, but hasn't impacted my Bruno experience up until this point.

**Reply by Its-treason** [reply]
That looks good, those two errors always happen in the production build.

One last idea: Is your collection saved on a network drive / device? This will also sometimes cause problem with the file watcher.

**Reply by andrewdavis12** [reply, marked as answer]
Everything is local.

I finally bit the bullet and blew away the collection. Starting a fresh collection had the same issue. So I had to go nuclear and completely wipe all traces of Bruno off the machine, reinstall to a different directory, and start everything from scratch. That seems to have solved it. Thanks for your suggestions though!

**Comment by rgdigital** [comment]
Same error here on Windows 11, although the installer doesn't allow me to install to another directory. Installs by default to -
C:\Users\<Username>\AppData\Local\Programs\bruno

Also tried installing via Choco. Restart is the only thing that brings up the newly-added resources in the UI.

**Comment by koolsing** [comment]
Happening in Bruno v3.3.0

**Comment by tjeufoolen** [comment]
Still seems to happen in Bruno v4.0.0. I've cleared my entire Bruno directory, but nothing seemed to fix it. Running on macos 26.6.1 and installed with homebrew. 

I also tried checking for updates manually, but it always shows 'No changes in collection'. Looking at the spec with 'view spec' shows something else then manually opening the source file.
